server/src/controller/answerController.js

```diff
@@ -9,17 +9,17 @@ const createAnswer = async (req, res) => {
       habit: req.body.habit,
     });
 
-    res.status(200).json(answer);
+    res.status(201).json(answer);
   } catch (err) {
-    res.status(400).json({ error: err.message });
+    res.status(500).json({ error: err.message });
```

Comment:
Thank you for your reply. I thought the error happened because of some kind of failure in database activity. That's why I suggested a 500 error code. This makes sense. I will update the codebase with the 400 error code.